test/integration/connect/envoy/case-mesh-to-lambda/service_defaults_l1.json

```diff
@@ -0,0 +1,11 @@
+{
+  "Kind": "service-defaults",
+  "Name": "l1",
+  "Protocol": "http",
+  "Meta": {
+    "serverless.consul.hashicorp.com/v1alpha1/lambda/enabled": "true",
+    "serverless.consul.hashicorp.com/v1alpha1/lambda/region": "${AWS_LAMBDA_REGION}",
+    "serverless.consul.hashicorp.com/v1alpha1/lambda/arn": "${AWS_LAMBDA_ARN}",
```

Comment:
Lambda configuration is injected as evironment variables and a variable expansion is performed here so that we don't expose information about our AWS account in our CI config.